spec/lib/tasks/provider_transfer/ptr_migrations_spec.rb

```diff
@@ -0,0 +1,179 @@
+require "rails_helper"
+
+RSpec.describe "ptr_migrations:", type: :task do
+  before do
+    Rails.application.load_tasks if Rake::Task.tasks.empty?
+  end
+
+  let(:initial_office) { create(:office) }
+  let(:new_office) { create(:office) }
+  let(:legal_aid_application) { create(:legal_aid_application, office: initial_office) }
+  let(:initial_firm) { create(:firm) }
+  let(:new_firm) { create(:firm) }
+  let(:provider) { create(:provider, firm: initial_firm) }
+  let(:firm) { create(:firm) }
+  let(:office) { create(:office) }
+
+  describe "ptr_migrations:laa_transfer_office" do
+    subject(:task) do
+      Rake::Task["ptr_migrations:laa_transfer_office"]
+    end
+
+    context "when LegalAidApplication and Office exist" do
+      it "transfers the LegalAidApplication to the new office" do
+        expect(Rails.logger).to receive(:info).with("LegalAidApplication (#{legal_aid_application.application_ref}) current office code: #{initial_office.code}")
+        expect(Rails.logger).to receive(:info).with("LegalAidApplication (#{legal_aid_application.application_ref}) successfully transferred to office code: #{new_office.code}")
+
+        task.execute(mock: "false", appid: legal_aid_application.id, officeid: new_office.id)
+
+        expect(legal_aid_application.reload.office).to eq(new_office)
+      end
+    end
+
+    context "when LegalAidApplication does not exist" do
```

Comment:
Is it worth adding an `expect` to verify that office doesn't change in these scenarios?

Reply:
@agoldstone93 @colinbruce I've updated relevant specs